packages/angular/projects/ui-angular/src/lib/components/authenticator/components/confirm-sign-up/confirm-sign-up.component.ts

```diff
@@ -37,22 +32,13 @@ export class ConfirmSignUpComponent {
   }
 
   public get confirmSignUpHeading(): string {
-    const { codeDeliveryDetails: { DeliveryMedium } = {} } = this.authenticator;
-    return DeliveryMedium === 'EMAIL'
-      ? translate('We Emailed You')
-      : DeliveryMedium === 'SMS'
-      ? translate('We Texted You')
-      : translate('We Sent A Code');
+    const { codeDeliveryDetails } = this.authenticator;
```

Comment:
Is `codeDeliveryDetails` a string? Why did we originally have `{ DeliveryMedium } = {}` on line 40?

Reply:
Because this text needed `DeliveryMedium` (with it defaulting to `{}`) to determine where Cognito sent the code to. Now that's being abstracted to the util.